app/src/sharedTest/java/org/oppia/android/app/walkthrough/WalkthroughTopicListFragmentTest.kt

```diff
@@ -92,6 +94,48 @@ class WalkthroughTopicListFragmentTest {
     )
   }
 
+  // It tests if progressBar gets hidden on scrolling to second row of items
```

Comment:
`// If progressBar is getting hidden on scrolling to the second row of items or not`

Reply:
Done